### PR TITLE
fixes #445 point 4 - Added G translate plugin

### LINKED ISSUE #445: Adding languages step by step
### Expected results
**Get together all! Show power of unity in all world including Asia. Make easily connected to all students and mentors...**
Adding languages around the world to site.
This issue is connected with this issue - https://github.com/fossasia/gci17.fossasia.org/issues/83

### Steps to reproduce the issue
- [ ] Translate installations files into more than 15 languages.
      - Sinhala added by @STEdirisuriya (Me)
      - Polish added by @kubami9  
      - French added by @Ploypiti 
- [ ] Translate paragraphs in index.html into more than 15 languages.
- [ ] Connect with jekyll for added languages into site. - [refer this]https://www.sylvaindurand.org/making-jekyll-multilingual/)
- [x] Until making to jekyll we have to give a method for translate it. So, add Google Translate Plug-in for site until it finished.
- [ ] Translate whole page into more than 10 languages.
- [ ] Add more languages into thank you contributing section.
- [ ] Add comments section for sharing ideas about site with more languages.
- [x] Add how to translate in README.md
## Would you like to work on this
Get together all! I am working on it with Sinhala language. Students help us for make translations. We will make it easier for adding translations. I need lot of help. This is a small step for big jump!

### PR DESCRIPTION
**Pull request by supunTE**
Please read and understand everything below
**Do not delete any text other than where you are instructed.**

**Students: If one of them is applicable to you. Please check it.**

Check by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a Preview link and screenshot showning after and before the changes.
- [ ] Images are `240 x 240` [w x h].
- [x] Included a description of change below.
- [x] Squashed the commits.

# Changes done in this Pull Request

- https://stedirisuriya.github.io/gci17.fossasia.org/#
- Fixes https://github.com/fossasia/gci17.fossasia.org/issues/445 point 4
- Until we make it to jekyll method translate we have to give a method for translate at least a bit. So I add Google Plugin there. I removed other language translate links because it is useless. In plugin there are all languages.

### CODE REVIEW

File: _includes/header.html [hunk 1]
```diff
@@ -108,29 +108,21 @@
...
-		                                </li>
-		                            </ul>
-		                        </li>
-		                    </ul>
-		                </div>
+						
+						<div class="module-group right">
+							<ul class="menu">
+								<div id="google_translate_element"></div>
+								<script type="text/javascript">
```
Comment (abishekvashok) [inline _includes/header.html]: Move the scripts to footer to reduce the impact of such scripts and make it async as well.

File: _includes/header.html [hunk 2]
```diff
@@ -108,29 +108,21 @@
...
+								</script>
+								<script type="text/javascript" src="//translate.google.com/translate_a/element.js?cb=googleTranslateElementInit"></script>
+							</ul>
+						</div>
 				</div>
 			</nav>
 		</div>
 
-		<div class="main-container">
+<div class="main-container">
```
Comment (abishekvashok) [inline _includes/header.html]: Why did you change the alignment

**Review by ParthS007 — CHANGES_REQUESTED** [review]
@STEdirisuriya Please do changes suggested  by @abishekvashok  and do proper alignment of plugin in header

**Comment by abishekvashok** [comment]
Close and open again to restart travis

**Review by akshatnitd — CHANGES_REQUESTED** [review]
Please sync your fork with the original repo properly. Changes made by others are coming in your PR.

**Comment by ParthS007** [comment]
@STEdirisuriya  Good work but can add it in header itself, It's not looking good below it 👍

**Comment by supunTE** [comment]
OK I add it on header

**Comment by harshit98** [comment]
@STEdirisuriya Have you done all the suggested changes?

@ParthS007 @akshatnitd Please review the pull request. 💯

**Comment by supunTE** [comment]
@harshit98 yes

File: index.html [hunk 3]
```diff
@@ -2,6 +2,8 @@
 ---
 {% include header.html %}
 
+<!--Google Translate Plugin-->
```
Comment (vaibhavsingh97) [inline index.html]: Why this comment?

Comment (supunTE) [reply index.html]: small mistake. This plugin was this place earlier before done requested

**Review by saarthakchats — APPROVED** [review]
Squash commits

**Review by saarthakchats — CHANGES_REQUESTED** [review]
Pls squash

**Comment by STEdirisuriya** [comment]
@saarthakchats I squashed it already. Think how many commits I made but there two only. I can't squash that merge commit If i squash it all those changes show here. i'll do it next time correctly. This time my work went wrong. Thank you!

**Review by isuruAb — APPROVED** [review]
yes it works correctly 👍 
screen shot 2017-12-30 at 7 45 33 am

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]
@STEdirisuriya There is no next time. Please squash your commit and use some meaningful commit message. Please change your PR title to `Fixes #<issue-number`: `<commit-message>`. Please follow best practices

**Review by vaibhavsingh97 — APPROVED** [review]

**Comment by vaibhavsingh97** [comment]
Thanks @STEdirisuriya for this wonderful PR. Keep it up 🏅
